.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.0.0
+current_version = 3.0.0-alpha
```

Comment:
Right, we definitely need to publish some alphas and betas before 3.0 final.

But I think it should be 3.0.0a0 (PEP 440).

Reply:
Will make that change